### DISCUSSION TITLE
Tiktok Company

### DISCUSSION THREAD

**Discussion by almeidaaray** [vercel/community#1153] [Help] [answered]
### Summary

Fui entrar no meu aplicativo e não consegui entrar mas cadê TD dinheiro q não saquei 

### Additional information

Quero receber o dinheiro q já tinha no aplicativo 

### Example

Sim

**Comment by amyegan** [comment, marked as answer]
If I understand correctly, you're having trouble collecting payment from TikTok. We are a different company and cannot help with that. 😞 

If you are running into problems with a third party service, you should contact that site's owner for help.

**Reply by amyegan** [reply]
Update: The site appears to be back online now! 😄 If you have any further problems, please contact the site owner for help 🙏
